CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+`ui/Scrollable` and `ui/useScroll` to avoid stuttering of Scroller content on drag in mobile devices
+`ui/Scroller`, `ui/VirtualList`, and `ui/useScroll` to calculate correctly `scrollLeft` on `androidChrome` platform for rtl locales
```

Comment:
It looks better to explain just 'android platform' rather than a specific internal UA name.